admin/database/database.go

```diff
@@ -465,9 +479,34 @@ type UpdateServiceOptions struct {
 
 // Usergroup represents a group of org members
 type Usergroup struct {
-	ID    string `db:"id"`
-	OrgID string `db:"org_id"`
-	Name  string `db:"name" validate:"slug"`
+	ID          string         `db:"id"`
+	OrgID       string         `db:"org_id"`
+	Name        string         `db:"name" validate:"slug"`
+	Description sql.NullString `db:"description"`
+	CreatedOn   time.Time      `db:"created_on"`
+	UpdatedOn   time.Time      `db:"updated_on"`
+}
+
+// UsergroupOrgRole represents an organization role of Usergroup
+type UsergroupOrgRole struct {
+	UsergroupID string `db:"usergroup_id"`
+	OrgName     string `db:"org_name"`
+	RoleName    string `db:"role_name"`
+}
+
+// UsergroupProjectRole represents a project role of Usergroup
+type UsergroupProjectRole struct {
+	UsergroupID string `db:"usergroup_id"`
+	ProjectID   string `db:"project_id"`
+	ProjectName string `db:"project_name"`
+	RoleName    string `db:"role_name"`
+}
```

Comment:
Similar to how `Member` is used for both org and project member users (we should rename the type to `MemberUser`, but it's okay not to do it in this PR to keep things simple), it would be nice if we could have one `MemberUsergroup` type used to represent a "member of type usergroup".